$OpenBSD$

Fix build with newlib supplied syscalls enabled.
acle-compat.h is libc/machine/arm.

Index: newlib/libc/sys/arm/Makefile.am
--- newlib/libc/sys/arm/Makefile.am.orig
+++ newlib/libc/sys/arm/Makefile.am
@@ -2,7 +2,7 @@
 
 AUTOMAKE_OPTIONS = cygnus
 
-INCLUDES = $(NEWLIB_CFLAGS) $(CROSS_CFLAGS) $(TARGET_CFLAGS)
+INCLUDES = $(NEWLIB_CFLAGS) $(CROSS_CFLAGS) $(TARGET_CFLAGS) -I$(newlib_basedir)/libc/machine/arm
 
 AM_CCASFLAGS = $(INCLUDES)
 
